pgjdbc/src/main/java/org/postgresql/jdbc/TimestampUtils.java

```diff
@@ -1110,7 +1119,7 @@ public LocalTime toLocalTimeBin(byte[] bytes) throws PSQLException {
       micros = ByteConverter.int8(bytes, 0);
     }
 
-    return LocalTime.ofNanoOfDay(micros * 1000);
+    return LocalTime.ofNanoOfDay(Math.multiplyExact(micros, 1000L));
```

Comment:
I adapted this like in my own code as it's better code style

Reply:
Sorry it is not really "code style", it is more about safety. All java.time code uses multiplyExact to prevent overflow errors, so we should do this here, too. But if you disagree, I can undo this line of code.

Reply:
I do not disagree with it, was just trying to understand the comment.